management/src/main/java/org/kaazing/gateway/management/monitoring/configuration/impl/DefaultMMFMonitoringDataManagerStub.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
As described above, MonitoringDataManagerInjectorImpl is used from ManagementGatewayListener->startingGateway() method.